portfolio/src/main/webapp/script.js

```diff
@@ -13,11 +13,14 @@
 // limitations under the License.
 
 /**
- * Toggles the drop down text.
+ * Toggles the dropdown text and dropdown arrow.
  * @param {string} id The id of the dropdown content to be toggled.
+ * @param {!Event} event The click event on a dropdown bar.
  */
-function toggleDropDown(id) {
-  document.getElementById(id).classList.toggle("show");
+function toggleDropDown(id, event) {
+  document.getElementById(id).classList.toggle('show');
```

Comment:
Shouldn't you. be assigning this to some variable?

Reply:
Or is this leftover code.

Reply:
Assign what? 
So the function is called when the dropdown bar is clicked. The arrow at the right of the bar will be changed from pointing down to pointing up.